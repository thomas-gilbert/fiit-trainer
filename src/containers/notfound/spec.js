import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './index.js';

describe('NotFound Component', () => {
	it('renders correctly', () => {
	  const component = renderer.create(
	    <NotFound />
	  ).toJSON();
	  expect(component).toMatchSnapshot();
	});
});