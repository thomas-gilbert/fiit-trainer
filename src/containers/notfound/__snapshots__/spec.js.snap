// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component renders correctly 1`] = `
<div
  className="not-found"
>
  <h3>
    Sorry, page not found!
  </h3>
</div>
`;
